### PR TITLE
Rewrite submission page

### PR DESCRIPTION
**Pull request by pishifat**
this page was a mess and now it is not a mess

---

### CODE REVIEW

File: wiki/Submission/en.md [hunk 1]
```diff
@@ -1,150 +1,31 @@
...
-
-![Heart icon](/wiki/shared/icon/heart.gif "Heart icon") Ranked beatmaps are moved to the [Ranked/Approved Beatmaps](https://osu.ppy.sh/community/forums/14) subforum, added to the [Ranked Beatmap listing](https://osu.ppy.sh/p/beatmaplist), included in the Ranked [Beatmap packs](https://osu.ppy.sh/p/packlist), and appear in their profile. The scores achieved when playing them count towards a player's Total Ranked Score in the [Player Rankings](https://osu.ppy.sh/p/playerranking) and the ratings players can give them count towards their Beatmapper Rank.
-
-![Beatmap Submission System Warning screen](BSS_Base.jpg "Beatmap Submission System Warning screen")
+Beatmaps can be uploaded to the osu! website via the in-game editor. Uploading a beatmap allows the beatmap to receive attention from other users and potentially enter the Ranked or Loved beatmap categories. 
 
 ## Submission
```
Comment (cl8n) [inline wiki/Submission/en.md]: remove this

File: wiki/Submission/en.md [hunk 2]
```diff
@@ -1,150 +1,31 @@
...
+If the beatmap a user is uploading is not already on the osu! website, the BSS window will state how many new uploads a user can publish. If the beatmap is nominated, the window warns that the nomination will be reset by updating the beatmap. If the beatmap is in the graveyard, the window warns that the beatmap will be raised to the pending beatmaps category.
 
-#### ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought Bubble
+![Beatmap Submission System warning screen](bss_warning.jpg "Beatmap Submission System warning screen")
 
-Usually called "Bubble" for short, this symbol is given when **a Beatmap Nominator is completely satisfied with every aspect of the beatmap and nominates it for ranking.** Once a second Beatmap Nominator approves of the beatmap in the same way, it is ![Heart icon](/wiki/shared/icon/heart.gif "Heart icon") Qualified. A Thought Bubble might be downgraded to a ![Star icon](/wiki/shared/icon/star.gif "Star icon") Star in case a second moderator detects an oversight on the first moderator's part, but no further downgrading can ever occur.
+### Submission Options
```
Comment (cl8n) [inline wiki/Submission/en.md]:
```suggestion
## Submission options
```

File: wiki/Submission/en.md [hunk 3]
```diff
@@ -1,150 +1,31 @@
...
 
-This is NOT to be confused with a "Kudosu Star". A Kudosu Star is given out by modders to maps that they think are good. Kudosu Stars give out varying priority points depending on who gives them, but they can place a map higher in the priority list. Please see [Kudosu](/wiki/Glossary/#kudosu) for information more information on Kudosu.
+Two checkboxes appear near the bottom of the window. The first says `Receive email notification on reply`, which adds the beatmap to the user's [modding watchlist](https://osu.ppy.sh/beatmapsets/watches). The second says `Load in browser after submission`, which will load the beatmap's listing page.
 
-#### No symbol
+![Beatmap Submission System uploading screen](bss_submitting.jpg "Beatmap Submission System uploading screen")
 
-All beatmaps have no symbol assigned to them when they are first submitted. A beatmap nominator thinks that the beatmap has some issues that prevent him/her from giving it a ![Star icon](/wiki/shared/icon/star.gif "Star icon") Star. This state shouldn't worry you *too much*, as the beatmap can usually be marked with a Star or even a Thought Bubble after the BN's suggestions are followed.
+### Limitations
```
Comment (cl8n) [inline wiki/Submission/en.md]:
```suggestion
## Limitations
```

File: wiki/Submission/en.md [hunk 4]
```diff
@@ -1,150 +1,31 @@
...
-
-![Heart icon](/wiki/shared/icon/heart.gif "Heart icon") Ranked beatmaps are moved to the [Ranked/Approved Beatmaps](https://osu.ppy.sh/community/forums/14) subforum, added to the [Ranked Beatmap listing](https://osu.ppy.sh/p/beatmaplist), included in the Ranked [Beatmap packs](https://osu.ppy.sh/p/packlist), and appear in their profile. The scores achieved when playing them count towards a player's Total Ranked Score in the [Player Rankings](https://osu.ppy.sh/p/playerranking) and the ratings players can give them count towards their Beatmapper Rank.
-
-![Beatmap Submission System Warning screen](BSS_Base.jpg "Beatmap Submission System Warning screen")
+Beatmaps can be uploaded to the osu! website via the in-game editor. Uploading a beatmap allows the beatmap to receive attention from other users and potentially enter the Ranked or Loved beatmap categories.
```
Comment (cl8n) [inline wiki/Submission/en.md]:
```suggestion
[Beatmaps](/wiki/Beatmaps) can be submitted to the osu! website via the [in-game editor](/wiki/Beatmap_Editor). Submitting a beatmap allows it to receive attention from other users and potentially enter the [Ranked](/wiki/Beatmaps#ranked) or [Loved](/wiki/Beatmaps#loved) categories.
```
uploaded => submitted just so we use the article title here, and link new terms

File: wiki/Submission/en.md [hunk 5]
```diff
@@ -1,150 +1,31 @@
...
--   ![Star icon](/wiki/shared/icon/star.gif "Star icon") Star
--   No symbol
--   ![Heart-broken icon](/wiki/shared/icon/broken-heart.gif "Heart-broken icon") Heart-broken
--   ![Nuke icon](/wiki/shared/icon/nuke.gif "Nuke icon") Nuke
-    -   *Graveyard* if inactive for a month.
-
-#### ![Heart icon](/wiki/shared/icon/heart.gif "Heart icon") Heart
-
-Once a second Beatmap Nominator nominates a beatmap with a ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought Bubble symbol, he/she qualifies it and it is automatically given this symbol when moved to the Ranked/Approved Beatmaps forum.
+`Upload Beatmap...` from the `File` dropdown in the editor (shortcut: `Ctrl` + `Shift` + `U`) will open the "Beatmap Submission System" (BSS) window. This initially displays lists resources for users to ensure their maps are suitable for ranking, ask mapping-related questions, and find feedback for their beatmaps.
```
Comment (cl8n) [inline wiki/Submission/en.md]:
```suggestion
Selecting `Upload Beatmap...` from the `File` dropdown in the editor (shortcut: `Ctrl` + `Shift` + `U`) will open the **Beatmap Submission System** (***BSS***) window. This initially lists resources to help users ask mapping-related questions, find feedback for their beatmaps, and ensure their maps are suitable for ranking.
```
parallelism isn't too great here but u can get away with it if u put the irregular phrase last 💭 
and bolded the main topic of article

File: wiki/Submission/en.md [hunk 6]
```diff
@@ -1,150 +1,31 @@
...
 
-#### ![Fire icon](/wiki/shared/icon/flame.gif "Fire icon") Flame
+### Warning Screen
```
Comment (cl8n) [inline wiki/Submission/en.md]: i dont think this needs to be its own section, u already talk about part of this screen in the prev paragraph

just remove the header and it works fine

File: wiki/Submission/en.md [hunk 7]
```diff
@@ -1,150 +1,31 @@
...
 
-A Flame is given to beatmaps that do not have the technical problems that ![Nuke icon](/wiki/shared/icon/nuke.gif "Nuke icon") Nuked beatmaps have, and it is considered as Approved beatmaps because they are long and fit within the Ranking Criteria. Older Approval criteria included total score or difficulty, but only length is taken into account now. Those maps are typically difficult and made as a challenge to top players.
+If the beatmap a user is uploading is not already on the osu! website, the BSS window will state how many new uploads a user can publish. If the beatmap is nominated, the window warns that the nomination will be reset by updating the beatmap. If the beatmap is in the graveyard, the window warns that the beatmap will be raised to the pending beatmaps category.
 
-#### ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought Bubble
+![Beatmap Submission System warning screen](bss_warning.jpg "Beatmap Submission System warning screen")
```
Comment (cl8n) [inline wiki/Submission/en.md]: "Beatmap Submission System screen" should be good enough for what it's showing
and move this above the `Upload beatmap...` paragraph
and also put the image into an `img/` folder here + change link
and also it's .png not jpg

File: wiki/Submission/en.md [hunk 8]
```diff
@@ -1,150 +1,31 @@
...
+The `Creator's Words` section allows for user input that appears alongside the beatmap's website listing page. This supports `BBCode` formatting.
 
-This is NOT to be confused with a "Kudosu Star". A Kudosu Star is given out by modders to maps that they think are good. Kudosu Stars give out varying priority points depending on who gives them, but they can place a map higher in the priority list. Please see [Kudosu](/wiki/Glossary/#kudosu) for information more information on Kudosu.
+Two checkboxes appear near the bottom of the window. The first says `Receive email notification on reply`, which adds the beatmap to the user's [modding watchlist](https://osu.ppy.sh/beatmapsets/watches). The second says `Load in browser after submission`, which will load the beatmap's listing page.
 
-#### No symbol
+![Beatmap Submission System uploading screen](bss_submitting.jpg "Beatmap Submission System uploading screen")
```
Comment (cl8n) [inline wiki/Submission/en.md]: img/ + move to right below the submission options header + jpg=>png

File: wiki/Submission/en.md [hunk 9]
```diff
@@ -1,150 +1,31 @@
...
-
-### How many maps may I upload?
-
-All osu! users have a maximum number of unranked beatmaps in pending they are allowed to have.
-
-Depending on how many ranked maps you have, you are allowed between **3 to 6 unranked maps.** This number increases by one each time one of your maps is ranked. Only maps in Pending and Help forums are counted. The cap is in place to help control the flow of new beatmaps and to keep mappers more dedicated to their existing maps.
+Beatmaps will fail to submit if they exceed the online file size limit. That limit is 5MB plus an additional 10MB for every minute of beatmap length, and it caps at 100MB.
 
-**osu!supporters** have a larger cap and are allowed between **4 to 10 unranked maps.**
+Users can have a limited number of pending beatmaps at a time. The limit varies depending on how many ranked beatmaps a user has and whether or not they currently have [osu!supporter](https://osu.ppy.sh/help/wiki/osu!supporter). Users without osu!supporter can have 4 pending beatmaps plus 1 per ranked beatmap (up to 2). With osu!supporter, this increases to 8 pending beatmaps plus 1 per ranked beatmap (up to 12).
```
Comment (dorothy3242) [inline wiki/Submission/en.md]: This is wrong, supporter's max is 20 now. I don't know the particulars of how non-supporter or number of ranked beatmaps currently plays into this though.

File: wiki/Submission/en.md [hunk 10]
```diff
@@ -1,150 +1,31 @@
...
+### Submission Options
 
-#### ![Star icon](/wiki/shared/icon/star.gif "Star icon") Star
+When clicking the `Update Beatmap!` button, a user will be presented with the choice of uploading their beatmap to the `Work In Progress (WIP)/Help` or `Pending Beatmaps` categories. WIP beatmaps cannot be nominated for ranking, while pending beatmaps can.
 
-A Star is given to a beatmap **a Beatmap Nominator judges to be on the right track but still needing a few more improvements before he or she approves of it with a ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought Bubble.** Examples include fairly minor adjustments in timing, incorrect Beat Spacing in some areas, or needing to create additional difficulty versions.
+The `Creator's Words` section allows for user input that appears alongside the beatmap's website listing page. This supports `BBCode` formatting.
```
Comment (dorothy3242) [inline wiki/Submission/en.md]: I think it'd be helpful to link BBCode to https://osu.ppy.sh/help/wiki/BBCode

**Comment by cl8n** [comment]
I'm not sure beatmap ranking procedure fits under Submission at all rly 

we need to figure out a better way to group pages about modding/ranking

File: wiki/Submission/en.md (hunk 9 above)
Comment (pishifat) [reply wiki/Submission/en.md]: 8 pending + 12 ranked map bonuses is 20? am i misinterpreting something

Comment (dorothy3242) [reply wiki/Submission/en.md]: O. The way it's  written comes off as 8 going up to a max of 12 (so capping after 4 ranked maps)

for a total of up to 20?

**Comment by cl8n** [comment]
all of these code styles for menu items look silly but ig this is how other articles do it

**Pull request merged**